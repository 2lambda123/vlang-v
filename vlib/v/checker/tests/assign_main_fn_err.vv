fn main() {
	a := main
	a()

	b := [main]
	println(b)
}
