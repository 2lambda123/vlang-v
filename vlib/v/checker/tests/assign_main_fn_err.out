vlib/v/checker/tests/assign_main_fn_err.vv:2:7: error: `main` function cannot be used anywhere
    1 | fn main() {
    2 |     a := main
      |          ~~~~
    3 |     a()
    4 |
vlib/v/checker/tests/assign_main_fn_err.vv:5:8: error: `main` function cannot be used anywhere
    3 |     a()
    4 |
    5 |     b := [main]
      |           ~~~~
    6 |     println(b)
    7 | }
